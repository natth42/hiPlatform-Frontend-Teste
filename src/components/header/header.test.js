import React from 'react';
import Header from './header';
import { shallow } from 'enzyme';

it('renders Header without crashing', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
});




