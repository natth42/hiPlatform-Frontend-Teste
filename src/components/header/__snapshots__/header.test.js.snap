// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <Navbar
        className="header-space"
        color="light"
        expand="md"
        light={true}
        tag="nav"
      >
        <NavbarBrand
          tag="a"
        >
          <h4
            id="title"
          >
            <span
              aria-label="pequena estrela"
              role="img"
            >
              ðŸ‘‹
            </span>
             Hi Platform
          </h4>
        </NavbarBrand>
      </Navbar>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <NavbarBrand
          tag="a"
        >
          <h4
            id="title"
          >
            <span
              aria-label="pequena estrela"
              role="img"
            >
              ðŸ‘‹
            </span>
             Hi Platform
          </h4>
        </NavbarBrand>,
        "className": "header-space",
        "color": "light",
        "expand": "md",
        "light": true,
        "tag": "nav",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <h4
            id="title"
          >
            <span
              aria-label="pequena estrela"
              role="img"
            >
              ðŸ‘‹
            </span>
             Hi Platform
          </h4>,
          "tag": "a",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <span
                aria-label="pequena estrela"
                role="img"
              >
                ðŸ‘‹
              </span>,
              " Hi Platform",
            ],
            "id": "title",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "aria-label": "pequena estrela",
                "children": "ðŸ‘‹",
                "role": "img",
              },
              "ref": null,
              "rendered": "ðŸ‘‹",
              "type": "span",
            },
            " Hi Platform",
          ],
          "type": "h4",
        },
        "type": [Function],
      },
      "type": [Function],
    },
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <Navbar
          className="header-space"
          color="light"
          expand="md"
          light={true}
          tag="nav"
        >
          <NavbarBrand
            tag="a"
          >
            <h4
              id="title"
            >
              <span
                aria-label="pequena estrela"
                role="img"
              >
                ðŸ‘‹
              </span>
               Hi Platform
            </h4>
          </NavbarBrand>
        </Navbar>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <NavbarBrand
            tag="a"
          >
            <h4
              id="title"
            >
              <span
                aria-label="pequena estrela"
                role="img"
              >
                ðŸ‘‹
              </span>
               Hi Platform
            </h4>
          </NavbarBrand>,
          "className": "header-space",
          "color": "light",
          "expand": "md",
          "light": true,
          "tag": "nav",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <h4
              id="title"
            >
              <span
                aria-label="pequena estrela"
                role="img"
              >
                ðŸ‘‹
              </span>
               Hi Platform
            </h4>,
            "tag": "a",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <span
                  aria-label="pequena estrela"
                  role="img"
                >
                  ðŸ‘‹
                </span>,
                " Hi Platform",
              ],
              "id": "title",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "aria-label": "pequena estrela",
                  "children": "ðŸ‘‹",
                  "role": "img",
                },
                "ref": null,
                "rendered": "ðŸ‘‹",
                "type": "span",
              },
              " Hi Platform",
            ],
            "type": "h4",
          },
          "type": [Function],
        },
        "type": [Function],
      },
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
